api/src/app/modules/forest-client/forest-client.service.ts

```diff
@@ -33,6 +42,44 @@ export class ForestClientService extends DataReadOnlyService<ForestClient, Repos
     return result.map(forestClient => this.convertEntity(forestClient));
   }
 
+  /**
+   * Forest_Client data refreshing to app_fom.forest_client table.
+   */
+  async batchClientDataRefresh(): Promise<void> {
+    this.logger.info(`Starting batch data refreshing on ${DateTimeUtil.nowBC()} for app_fom.forest_client records...`);
+    let fetchedData: Array<ClientAppIntegrationResponse> = [];
+    let currentPage = 0;
+    let totalRecordsCount = 0;
+
+    do {
+      fetchedData = await this.clientAppIntegrationService.fetchClientNonIndividuals(
+        currentPage, 
+        this.configService.get("fcApiBatchSerchPageSize"),
+        ClientAppIntegrationService.SORT_BY_CLIENT_NUMBER
+      );
+
+      fetchedData.forEach(async (item) => {
+        let entity = await this.repository.findOne({ where: { id: item.id } })
+        if (!entity) {
+          entity = mapToEntity(item, new ForestClient());
+          entity.createUser = USER_SYSTEM;
```

Comment:
Where do we set the create timestamp?

Reply:
The db in fact has a default value for column `create_timestamp` but not on update_timestamp. So only `update` needs to set the time.